velox/functions/prestosql/SIMDJsonFunctions.h

```diff
@@ -163,6 +163,48 @@ struct SIMDJsonArrayLengthFunction {
   }
 };
 
+template <typename TExec>
+struct SIMDJsonArrayGetFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExec);
+
+  FOLLY_ALWAYS_INLINE bool
+  call(out_type<Json>& result, const arg_type<Json>& jsonArray, int64_t index) {
+    simdjson::ondemand::document jsonDoc;
+
+    simdjson::padded_string paddedJson(jsonArray.data(), jsonArray.size());
+    if (simdjsonParse(paddedJson).get(jsonDoc)) {
```

Comment:
`simdjsonParse(paddedJson).get(jsonDoc) != simdjson::error_code::SUCCESS` for readability

Reply:
This adds quite a bit of code and doesn't align with what simdjson docs recommend: https://github.com/simdjson/simdjson/blob/master/doc/basics.md#error-handling